lib/instrumental/agent.rb

```diff
@@ -77,10 +78,15 @@ def initialize(api_key, options = {})
       @certs           = certificates
       @dns_resolutions = 0
       @last_connect_at = 0
+      @metrician       = options[:metrician].nil? ? true : !!options[:metrician]
```

Comment:
I just followed the common pattern above because I didn't feel like introducing more change, but we can change all of them if you want.